src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__link {
+  margin: 8px 0 8px 50px;
+
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style-type: none;
+}
+
+.nav__link {
+  padding: 22.5px 0;

```

Comment:
For vertical centering the text, you could use display: flex and align-items: center.
PAdding with a such strange numbers are not good

Reply:
I use paddink here for add activ area to link